default_config/dev/config.yaml

```diff
@@ -1,8 +1,19 @@
 API_HOST_URL: 'http://localhost:8000'
+SSO_URL: 'http://localhost:8000'
+SSO_URL_SUCCESSFUL_LOGIN:  'http://localhost:8000'
 ENVIRONMENT_DISPLAY_NAME: 'Localhost'
 FEEDBACK_EMAIL_ADDRESS: 'support@example.com'
 OAUTH_CLIENT_ID: ''
 
+SSO_BUTTON_TITLE: {
+ar: "الدخول عبر SSO",
+en: "Sign in with SSO"
+}
+
+
 UI_COMPONENTS:
     COURSE_UNIT_PROGRESS_ENABLED: false
-    COURSE_NESTED_LIST_ENABLED: false
+    COURSE_NESTED_LIST_ENABLED: true
+    LOGIN_REGISTRATION_ENABLED: true
+    SAML_SSO_LOGIN_ENABLED: true
+    SAML_SSO_DEFAULT_LOGIN_BUTTON: false
```

Comment:
Please do not change the default behavior of feature flags.

Reply:
✅